packages/card/src/Card.tsx

```diff
@@ -28,10 +28,10 @@ const lightHoverBoxShadow = `0 4px 20px -4px ${transparentize(
   0.8,
   palette.black,
 )}`;
-const lightFocusBoxShadow = focusRing['light'].default;
-const darkBaseBoxShadow = `0 4px 10px -4px  ${transparentize(0.4, '#000D13')}`;
+const lightFocusBoxShadow = focusRing.light.default;
+const darkBaseBoxShadow = `0 4px 20px -4px #01121A`;
```

Comment:
No transparency?

Reply:
yeah according to Figma

Reply:
Maybe add a comment for the next person who looks at this?